src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Apprentice/Dates.cshtml

```diff
@@ -83,5 +84,12 @@
 
 @section breadcrumb
 {
-    <a id="back-link" class="govuk-back-link" href="@Url.RouteUrl(RouteNames.ApprenticeConfirmEmployer, new { Model.EmployerAccountLegalEntityPublicHashedId})">Back</a>
+    @if (Model.InEditMode)
+    {
+        <a id="back-link" class="govuk-back-link" href="@Url.RouteUrl(RouteNames.ApprenticeConfirmChangeOfEmployer, new { Model.EmployerAccountLegalEntityPublicHashedId, Model.StartDate.MonthYear, Model.Price })">Back</a>
```

Comment:
Using this link gives an error: `{"StartDate":["The specified condition was not met for 'Start Date'."]}`

Reply:
bound to the wrong name, fixed